triage/templates/triage/start-now.html

```diff
@@ -0,0 +1,36 @@
+{% extends "core/base.html" %}
+
+{% load static from staticfiles %}
+
+{% block head_title %}
+    Your export journey - great.gov.uk
+{% endblock %}
+
+{% block css %}
+    <link href="{% static 'styles/custom-page.css' %}" rel="stylesheet" type="text/css" />
+{% endblock %}
+
+{% block content %}
+	<div class="container">
+		<div  id="start-now-container">
+			<h1>Create your export journey</h1>
+			<br>
+			<p>
+				Answer a few quick questions about what you want to export and your level of experience and get:
+				<ul>
+					<li>guidance aimed at your experience level</li>
```

Comment:
can we use spaces instead of tabs here? all the other templates seem to use spaces